addons/account/views/report_invoice.xml

```diff
@@ -249,10 +249,15 @@
                     </p>
                     <div id="qrcode" t-if="o.display_qr_code and o.amount_residual > 0">
                         <t t-set="qr_code_url" t-value="o._generate_qr_code()"/>
-                        <p t-if="qr_code_url">
-                            <strong class="text-center">Scan me with your banking app.</strong><br/><br/>
-                            <img class="border border-dark rounded" t-att-src="qr_code_url"/>
-                        </p>
+                        <div t-if="qr_code_url" class="col-6">
+                            <div class="float-start">
+                                <strong class="text-center">Scan me with your banking app.</strong>
+                                <div id="qr_info"/>
```

Comment:
The problem is that if you do not update account, the installation of UPI will fail.  Is not there another way to do it?

Reply:
No problem if we improve the view a bit in itself, but maybe we can search the strong instead?

Reply:
this is in the master so no need to worry about that one.
it's auto-update In-migration

Reply:
oh yeah, master